src/cursor_on_target.py

```diff
@@ -27,31 +27,82 @@
 import uuid
 import time
 import socket
-#import struct
-import socket
 import logging
+import math
+import hashlib
+import argparse
+import config
+
+# NATO phonetic alphabet
+NATO_ALPHABET = [
+    "ALPHA", "BRAVO", "CHARLIE", "DELTA", "ECHO", "FOXTROT", "GOLF", "HOTEL", "INDIA",
```

Comment:
Typo in first code word, should be "Alfa" instead of "Alpha" (easier for non-native English speakers I guess?)

Also, would prefer that only first letter of each code word is capitalized